styles/mixins/row-carousel.css

```diff
@@ -0,0 +1,19 @@
+@define-mixin row-carousel {
```

Comment:
what you're not seeing is the helper file didn't change since it was a bunch of responsive classes calling the same mixin.  This prevents us from writing the same code for 4 different classes.